plutus-ledger-api/src/PlutusLedgerApi/V1/Scripts.hs

```diff
@@ -45,6 +45,7 @@ The `Serialise` instances for `Datum` and `Redeemer` exist for several reasons:
   have to import `BuiltinData` which is from a different package.
 -}
 
+-- | A higher-level evaluation error used by 'plutus-apps'
```

Comment:
Hmm, do we want to refer to `plutus-apps` here? I guess it's more just the kind of error returned by the functions in this module, that's about it ...

Reply:
I wanted to mention in here, because this datatype is kind of dead. It is not constructed/thrown anywhere in the plutus repository based on my grep'ing.

Reply:
I think it should be moved away from the api and into plutus-apps